### ISSUE TITLE
Impossible to override cookie

### ISSUE THREAD

**Issue by lostfields** [Huachao/vscode-restclient#156]
- VSCode Version: 1.19.2
- OS Version: W10Pro
- REST Client Version: 0.16.2

I have a problem where I'm unable to clear the stored cookies for subsequent requests. The API I'm communicating with doesn't allow the authenticated cookie be sent over another time when authenticating. So, I'm trying to not sending it over by defining Cookie header, but no luck. 

Empty cookie key is not allowed by the api either so I can't override it (like "Cookie: key=<empty>"). 

I have to ignore cookies for this request, is that possible?

**Comment by Huachao** [comment]
@lostfields currently the only possible way is to clear the stored cookie.json file under `~/.rest-client/cookie.json`

**Comment by Huachao** [comment]
Do you mean only use saved cookies
only when cookie header is not present in user's request

**Comment by lostfields** [comment]
yes or the possibility to turn off cookie for a certain request by using local variables/settings just for that request-block.

**Comment by james-klein** [comment]
+1 would love to be able to leave out the Cookie header. I tried doing something like this:

`POST https://myapi.com/x
Cookie: 

{
    "foo": "bar"
}`

hoping it would just not send a Cookie header at all, but instead it sent the saved cookie (as intended, I'm assuming).

**Comment by chixu** [comment]
Has this feature(clear the cookie in the subsequent requests) been implemented?

**Comment by chixu** [comment]
Cool saw it the setting..

**Comment by KrishnaPG** [comment]
Came across this while facing the problem of cookie header not being sent. It has been quite difficult to troubleshoot, since I have been trying to send a cookie header manually and the server kept rejecting it and no idea why. Finally this issue [here](https://github.com/Huachao/vscode-restclient/issues/284) has the location of the file and I had to manually delete it. And still there is no way to send a cookie header manually.
```
### Get All Roles
GET http://{{casdoorHost}}/api/get-all-roles
Cookie: casdoor_session_id={{casdoor_session_id}}
```

Hope to see this fixed. Explicitly sent headers such as above should always take precedence over the saved cookies.

**Comment by guysoft** [comment]
@Huachao Even just adding ``~/.rest-client/cookie.json`` as the cookie path to the docs is helpful. I can update that manually to what I need. Even script getting the cookies directly to it.
